book/tut04_geophysics.ipynb

```diff
@@ -0,0 +1,694 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "db1ca43e-7b27-4e48-aee1-646cd00f75bf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "region = [-90, -60, -40, 0]  \n",
+    "\n",
+    "grid = pygmt.xyz2grd(data=data, spacing=\"5m\", region=region)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c3e53ce1-20e4-4ede-8d88-fdd95350e77e",
+   "metadata": {},
+   "source": [
+    "We can get some information about the grid using [`pygmt.grdinfo`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.grdinfo.html):"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "729e8a9e-2896-426a-b321-468486597f80",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(pygmt.grdinfo(grid))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3d08000c-7ac9-4899-bd3f-2006280d02d1",
+   "metadata": {},
+   "source": [
+    "In PyGMT, the grid is stored in an `xarray.DataArray`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "493f5bba-0769-4d52-88a6-41cb79557b6f",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "grid"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b238bef-3018-4bdc-92f8-9caa888a2c32",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## 2. Create a contour map: [`pygmt.Figure.grdcontour`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.Figure.grdcontour.html)\n",
+    "To plot grids or images as contour lines you can use [`pygmt.Figure.grdcontour`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.Figure.grdcontour.html). You need to define:\n",
+    "\n",
+    "1. `grid`: Access the remote datasets or providing your dataset as `xarray.DataArray`.\n",
+    "2. `level`: Specify the contour intervals to generate; for example, a value of 1000 means plotting contour lines at every 1000 m.\n",
+    "3. `annotation`: Annotate contour levels.\n",
+    "4. `limit`: Draw contours below low or above high, e.g., [-4000, 0] means drawing contour lines below sea level and above -4000 m.\n",
+    "\n",
+    "Before we plot the grid as contour lines, we plot the grid with color-coding using [`pygmt.Figure.grdimage`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.Figure.grdimage.html). Via the `shading` parameter, we can apply an illumination to the grid or image, which can help to create visually appealing maps. By passing the argument **+a**, we use an azimuth of -45 degrees. Run the next code cell twice, first without and second with illumination, to see the difference. For more a complicated illumination, please refer to [`pygmt.grdgradient`](https://www.pygmt.org/v0.13.0/api/generated/pygmt.grdgradient.html), which was already introduced in Tutorial 3."
```

Comment:
I am wondering if and how it is possible to add links to the other Jupyter notebooks or scripts (if this is possible, maybe we can also do this in a common PR).

Reply:
I'm not sure I fully understand your point. Are you suggesting that this information could be linked to the current gallery?

Reply:
Hm. What do you mean with "current gallery"? 

We partly mention other tutorials, which is good. I feel it would be nice to directly insert the link. But we have the URL from GitHub (https://github.com/GenericMappingTools/agu24workshop/blob/main/book/tut0XYZ.ipynb) as well as the one from the  Jupyter book (https://www.generic-mapping-tools.org/agu24workshop/XYZ.html). So depending on which website people are, the link does lead them to another "base" website. As the Jupyter book website is the "official" website for this workshop and GitHub is more for development, maybe using the URL to the Jupyter book website makes more sense here?

Reply:
Now I understand what you mean. 
> As the Jupyter book website is the "official" website for this workshop and GitHub is more for development, maybe using the URL to the Jupyter book website makes more sense here?

YES. Providing the URL to the Jupyter Book website will make it more convenient and readable.